### DISCUSSION TITLE
Changelog? Semver?

### DISCUSSION THREAD

**Discussion by carueda** [outr/scribe#515] [General]
Hi there!

From to time I merge PRs from scala-steward in some of my scala projects. Besides a green CI, I also try to make sure there are no incompatibility issues of some sort. There are PR links to diffs and such, but this approach is far from ideal. In the case of scribe, I miss having a CHANGELOG, MIGRATION or similar file to double-check all would be fine with an upgrade or whether some extra work might be needed.  I also just searched for "semver" in this repo but did not find anything. Thanks in advance for any reactions. (if this is not a right place for this, feel free to delete this entry.)

**Comment by darkfrog26** [comment]
I'd welcome any suggestions for improvements. I'd especially welcome a PR if you have a suggested change to the code.

**Comment by carueda** [comment]
Thanks, appreciate the reaction. Don't get me wrong, I know that the release entries contain comments, which have certainly been useful). However, the title here was already intended as a suggestion: have some sort of [well-known file](https://keepachangelog.com) for summarizing the main changes, especially upon major or major.minor updates. This is not about code per se, and it is a suggestion for the main maintainers, who have the expertise to explain the changes.  Again, sorry for any noise here and feel free to close/cancel as you see fit.

**Comment by darkfrog26** [comment]
I'm not opposed to a changelog, but I try to make sure to include information about changes in the releases on GH (for example: https://github.com/outr/scribe/releases/tag/3.12.2). What you're suggesting about a changelog would be more beneficial for larger jumps in versions?
